### ISSUE TITLE
Error when POSIXct.out argument is FALSE

### ISSUE THREAD

**Issue by adokter** [adokter/suntools#10]
```
crepuscule(
       matrix(c(-76.4511, 42.4800), nrow = 1),
       as.POSIXct("2023-06-01", tz = "America/New_York"),
       solarDep = 6,
       direction = "dawn",
       POSIXct.out = FALSE
     )
     
 Error in 1:nrow(res) : argument of length 0
```

**Comment by iskandari** [comment]
Looks like there is also an issue with nrow > 1 matrices from the original function:

```r

set.seed(123)
m = cbind(runif(4, min = -180, max = 180), runif(4, min = -90, max = 90))

maptools::crepuscule(
  m,
  as.POSIXct("2023-06-01", tz = "America/New_York"),
  solarDep = 6,
  direction = "dawn",
  POSIXct.out = TRUE
)
#day_frac                time
#  1       NA                <NA>
#  2       NA                <NA>
#  3 0.149016 2023-06-01 03:34:34
#  4       NA                <NA>

suntools::crepuscule(
  m,
  as.POSIXct("2023-06-01", tz = "America/New_York"),
  solarDep = 6,
  direction = "dawn",
  POSIXct.out = TRUE
)
#day_frac                time
#  1       NA                <NA>
#  2       NA                <NA>
#  3 0.149016 2023-06-01 03:34:34
#  4       NA                <NA>
```

**Comment by iskandari** [comment]
fixed by #11 e5feedfce4d25e0890c37fae44fb1f7a2ba1a40d
